PostProcessing/IntegralsPlot.py

```diff
@@ -1,140 +1,64 @@
-    if not oneplot:
-
-        #iterate over the variables and plot them
-        for i in range(len(plot_variables)):
-            setup_figure()
-            plotvar = plot_variables[i]
-
-            if linestyles is not None:
-                linestyle = linestyles[i]
-            else:
-                linestyle = '-'
-
-            make_plot(data_obj, plotvar, plotbounds, linestyle)
-            plt.legend()
-            save_fig(plotpath, plotvar)
-
-    #combine all the plots into one
-    else:
-        setup_figure()
-        for i in range(len(plot_variables)):
-            plotvar = plot_variables[i]
-
-            if linestyles is not None:
-                linestyle = linestyles[i]
-            else:
-                linestyle = '-'
-            make_plot(data_obj, plotvar, plotbounds, linestyle)
-        plt.legend()
-        plt.ylabel("") #remove ylabel for combined plot
-        save_fig(plotpath, "combined")
+from Source.OneD import *
+from Source.Common.Utils import get_config
+
+## Load config file
+config = get_config(sys.argv) #includes error checking
+
+#if the plot and movie paths dont exist, create them
+create_output_dirs(config)
+
+#Instantiate printer function that accepts a verbosity level
+verbPrint = VerbosityPrint(config["Header"].getint("verbosity",0))
+verbPrint("Verbosity: ", verbPrint.verbosity)
 
 
 def main():
+    """
+    Main function to plot the data
+
+    Raises:
+        IOError: Data path not found
+        IOError: Data file not found
+        RuntimeError: No plot variables specified
+        SystemError: No plot variables specified
+        SystemError: Number of x limits does not match number of plot variables
+        SystemError: Number of linestyles does not match number of plot variables
+    """
+
+    #Extract basic file information
+    datapath = config["Header"].get("data_path", fallback = "./data")
+    filename = config["Header"].get("data_filename", fallback = "Integrals.dat")
 
     # get the filename that contains the data
     data_filename_abs = os.path.join(datapath, filename)
     verbPrint("Data file: {0}".format(data_filename_abs))
 
+    #check if data directory exists
+    if not os.path.exists(datapath):
+        raise IOError("Data path not found: {0}".format(datapath))
+
     # check if file exists, if not raise SystemError
     if not os.path.exists(data_filename_abs):
-        raise SystemError("Data file not found: {0}".format(data_filename_abs))
+        raise IOError("Data file not found: {0}".format(data_filename_abs))
     
     # get the file object
     data_obj = get_data_dataframe(data_filename_abs)
 
     #get the plot variables
-    plot_variables = config["VariableData"].get("plot_variables", "").split()    
-    #check the plot variables are non-empty
-    if not plot_variables:
-        raise SystemError("No plot variables specified.")
+    plot_variables = config["VariableData"].get("plot_variables", "").split()
+    if len(plot_variables) == 0:
```

Comment:
If you want, you can just write
```suggestion
    if plot_variables:
```
An empty list evaluates to false.

Reply:
Ah, yea thats right. I've been using so much C++ i forgot about those nice pythonic things. Changed in newest commit to this PR